### PR TITLE
Remove showing logic for not valid user or password. Connected to edenlabllc/ehealth.api/issues#1550

### PR DESCRIPTION
**Pull request by ivchenkodima**
https://github.com/edenlabllc/ehealth.api/issues/1550

### CODE REVIEW

File: app/common/containers/pages/SignInPage/redux.js [hunk 1]
```diff
@@ -19,15 +19,11 @@ dispatch(createSessionToken({
     const { message } = action.payload.response.error;
     if (message === 'User blocked.') {
       throw new SubmissionError({
-        email: { user_blocked: true },
+        email: { accountPasswordMismatch: true },
```
Comment (ynotLeft) [inline app/common/containers/pages/SignInPage/redux.js]: why do we throw account password mismatch error if user is blocked?

File: app/common/containers/pages/SignInPage/redux.js [hunk 2]
```diff
@@ -19,15 +19,11 @@ dispatch(createSessionToken({
...
       });
     } else if (message === 'Identity, password combination is wrong.') {
       throw new SubmissionError({
-        password: { passwordMismatch: true },
-      });
-    } else if (message === 'User not found.') {
-      throw new SubmissionError({
-        email: { identityMismatch: true },
+        email: { accountPasswordMismatch: true },
```
Comment (ynotLeft) [inline app/common/containers/pages/SignInPage/redux.js]: What is the difference between accountPasswordMismatch and passwordMismatch?

Comment (ivchenkodima) [reply app/common/containers/pages/SignInPage/redux.js]: AccountPasswordMismatch goes for not valid password or email. passwordMismatch - is only for password. So there is different text for them.

File: app/common/containers/pages/SignInPage/redux.js (hunk 1 above)
Comment (ivchenkodima) [reply app/common/containers/pages/SignInPage/redux.js]: Mistype, thx)

File: app/common/containers/pages/SignInPage/redux.js (hunk 2 above)
Comment (ynotLeft) [reply app/common/containers/pages/SignInPage/redux.js]: IMO there is too many names for this occurrence  it makes this situation confusing. 
Message says: 'identity', error is 'account' and in reality it is about email. Why don't make message 'Email or password is wrong' and error 'EmalOrPasswordMismatch' for example.

**Review by kiurchv — APPROVED** [review]

File: app/common/components/ErrorMessages/index.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ export default class ErrorMessagesTranslated extends React.Component {
         <ErrorMessage when="passwordMismatch">Не вірно введено пароль</ErrorMessage>
         <ErrorMessage when="identityMismatch">Такого користувача не існує</ErrorMessage>
 
-        <ErrorMessage when="accountPasswordMismatch">{t('Account, password combination is mismatch')}</ErrorMessage>
+        <ErrorMessage when="emailOrPasswordMismatch">{t('Account, password combination is mismatch')}</ErrorMessage>
```
Comment (ynotLeft) [inline app/common/components/ErrorMessages/index.js:31]: Account or password combination mismatch.

**Review by ynotLeft — APPROVED** [review]

**Pull request merged**